docs/src/1_Mechanics/01_PlayerGuide_Full.md

```diff
@@ -732,12 +779,27 @@ Skills.
   Experience Points to spend when they are added to the Character Sheet during
   Character Creation.
 
-- Powers are special features, boons or strengths, that cost 1 or 2 points. Powers may
-  have additional requirements such as Role or Level.
+- Powers are special features, boons or strengths, that cost 1 or 2 Experience Points.
+  If a Power has a Power Point cost, add the higher PP value to your total PP. Powers
+  may have additional requirements such as Role or Level.
 
 For a full set of character creation options, see our
 [Character Creation page](./03_CharacterCreation.md).
 
+### Leveling up
+
+As your character journeys through the world and accumulates experience, they will
+*Level Up*. Every character starts at Level 1 and gains experience over time.
+
+When leveling up, characters:
+
+- Add 1 to their maximum HP
```

Comment:
Is this the same as the point of HP that reflects level? If so, this wording implies an additional point.
```suggestion
```